target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -234,6 +235,7 @@
 
 #define RTL8367B_CPU_PORT_NUM		5
 #define RTL8367B_NUM_PORTS		8
+#define RTL8367B_NUM_PHYS		5
```

Comment:
The family of switches supported by rtl8367c includes 10 port switches (8+2ext). For the record, they are:
- RTL8363NB
- RTL8364NB
- RTL8365MB-VC
- RTL8367RB-VB
- RTL8367SB
- RTL8367S
- RTL8366SC
- RTL8363SC
- RTL8370MB
- RTL8310SR
- RTL8363NB-VB
- RTL8364NB-VB
- RTL8363SC-VB

Any switch that has more than 5 user ports will break this limit. The correct way to support multiple devices is to not use it as a constant.

For now, if it doesn't break current devices, we could stick with the constant but add a comment it might need to be revisited.

Reply:
You are right, this patch focuses on 5+2ext .For larger/smaller switches the code needs to be reworked.